### ISSUE TITLE
Training parameters for BTCUSD

### ISSUE THREAD

**Issue by MasanoriYamada** [issaz/sigker-nsdes#2]
Hello,

Thank you for the great project.

In the condition_nsde.ipynb
I changed the parameters to BTCUSD, 15M and when I run it, I am having trouble with the loss not going down.

It would be helpful if you could tell me what parameters you used to learn BTCUSD 15M.

**Comment by issaz** [comment]
Hi Masanori,

As mentioned in the Appendix of the associated paper, it's likely that your
scaling parameter of the conditional paths is not set correctly.

As a rule of thumb, you'd want the range of the (centred) terminal
distribution of the conditional paths to be approximately N(0,1). This
might require a bit of hyperparameter tuning.

All the best,

Zach

**Comment by MasanoriYamada** [comment]
Hi issaz,

As you mentioned, I changed the scale parameter and the learning rate and loss went down.

I tested it on other datasets and it seems to have worked well. Thanks for your great job!

**Comment by issaz** [comment]
Glad to hear it! Glad you have found it useful.
